### PR TITLE
qm-reboot: add page

### PR DESCRIPTION
**Pull request by byjujohn**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by byjujohn** [comment]
Refers #8537

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/linux/qm-reboot.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# qm reboot
+
+> Reboot a virtual machine by shutting it down, and starting it again after applying pending changes.
+> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- Reboot a virtual machine:
+
+`qm reboot {{VM_ID}}`
```
Comment (navarroaxel) [inline pages/linux/qm-reboot.md:8]: We use `snake_case` for placeholders
```suggestion
`qm reboot {{vm_id}}`
```

File: pages/linux/qm-reboot.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+- Reboot a virtual machine after wait for at most 10 seconds:
+
+`qm reboot --timeout {{10}} {{VM_ID}}`
```
Comment (navarroaxel) [inline pages/linux/qm-reboot.md:12]:
```suggestion
`qm reboot --timeout {{10}} {{vm_id}}`
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**